backend/app/views/spree/admin/return_authorizations/index.html.erb

```diff
@@ -3,7 +3,7 @@
 <% content_for :page_actions do %>
   <% if @order.shipments.any? &:shipped? %>
     <li>
-      <% if can? :create, Spree::ReturnAuthorization %>
+      <% if can? :manage, Spree::ReturnAuthorization %>
```

Comment:
same question as above

Reply:
@kennyadsl Same comment with the OrderDisplay issue mentioned above.